webservices/tests/candidate_tests.py

```diff
@@ -96,22 +95,11 @@ def test_fields(self):
 
     def test_extra_fields(self):
         response = self._results('/candidate/P80003338?year=2008')
-        self.assertIn('committees', response[0])
         self.assertIn('PO BOX 8102', response[0]['address_street_1'])
         self.assertIn('60680',response[0]['address_zip'])
         # testing for year sensitivity
         self.assertIn('O', response[0]['incumbent_challenge'])
 
-    def test_candidate_committes(self):
-        response = self._results('/candidate/P80003338?year=*')
```

Comment:
A couple questions:
1. Is this territory well-covered by a test for `candidate/P80003338/committees`?
2. If not, can this test be morphed into one for that^^ without too many changes?

If it's already covered or it would be too much work, we can leave it for when we work on the other endpoint, in another issue.